app/code/Magento/Authorization/Model/Rules.php

```diff
@@ -19,6 +19,8 @@
  * @method \Magento\Authorization\Model\Rules setAssertId(int $value)
  * @method string getPermission()
  * @method \Magento\Authorization\Model\Rules setPermission(string $value)
+ * @method string[] getResources()
+ * @method \Magento\Authorization\Model\Rules setResources(string[] $resource)
```

Comment:
Should be `resources`, I believe? Please squash changes into single commit and force push.